### PR TITLE
docs(manual): update english documentation

### PR DESCRIPTION
**Pull request by recaph**
english documentation for about, getting started, faq, upgrade, contact, concepts-overview,shape-attrs

##### Checklist

- [x] commit message follows commit guidelines

##### Description of change

Update English documentation pages for about, getting started, faq, upgrade, contact, concepts-overview and shape-attrs

### CODE REVIEW

File: docs/manual/about-g2.en.md [hunk 1]
```diff
@@ -1,32 +1,32 @@
 ---
-title: G2 简介
+title: Introduction to G2
```
Comment (simaQ) [inline docs/manual/about-g2.en.md:2]: 这篇文档，建议和 G2 GitHub repo 的 README 统一一下

File: docs/manual/concepts-overview.en.md [hunk 2]
```diff
@@ -1,94 +1,94 @@
...
-title: G2 基础概念概览
+title: G2 basic concepts overview
 order: 2
 ---
 
-为了更好得使用 G2 进行数据可视化，我们需要了解 G2 的图表组成以及相关概念。
+In order to better use G2 for data visualization, we need to understand the composition of G2 charts and related concepts.
 
-## 图形语法
+## Graphic grammar
```
Comment (simaQ) [inline docs/manual/concepts-overview.en.md:8]: The Grammar of graphics，一般直接同那么图形语法书的书名保持一致

Comment (recaph) [reply docs/manual/concepts-overview.en.md]: text updated. regards.

File: docs/manual/about-g2.en.md (hunk 1 above)
Comment (recaph) [reply docs/manual/about-g2.en.md]: page updated to match readme file. regards.

**Review by simaQ — APPROVED** [review]

**Pull request merged**